bdsx/bds/packets.ts

```diff
@@ -674,16 +674,22 @@ export class CameraPacket extends Packet {
 export class BossEventPacket extends Packet {
     @nativeField(bin64_t)
     unknown:bin64_t;
-    @nativeField(bin64_t)
-    entityUniqueId:bin64_t;
-    @nativeField(bin64_t)
-    unknown2:bin64_t;
-    @nativeField(uint32_t)
+    @nativeField(bin64_t, 0x38)
```

Comment:
For easy maintenance, Please reduce to use of offsets if it's clear.
The size of `Packet` is 0x30 and bin64_t is 8bytes.
So it's 0x38 clearly without the offset value.

Reply:
ok